spinc/cmd/help.go

```diff
@@ -85,13 +85,14 @@ func (c *Help) Usage() {
 		"  --help     Print help\n"+
 		"  --timeout  API timeout, milliseconds (default: %d)\n"+
 		"  --version  Print version\n"+
```

Comment:
Would be nice to still have an empty line between Flags + Commands (aka add `\n` to the end of this line).

Reply:
There's one: it's on next line: `"\nRun spinc (no command) to lists requests\n"`